lib/internal/queue_microtask.js

```diff
@@ -3,37 +3,36 @@
 const { ERR_INVALID_ARG_TYPE } = require('internal/errors').codes;
 const { AsyncResource } = require('async_hooks');
 const { getDefaultTriggerAsyncId } = require('internal/async_hooks');
-const { enqueueMicrotask } = internalBinding('util');
+const {
+  enqueueMicrotask,
+  triggerFatalException
+} = internalBinding('util');
 
-const setupQueueMicrotask = (triggerFatalException) => {
-  const queueMicrotask = (callback) => {
-    if (typeof callback !== 'function') {
-      throw new ERR_INVALID_ARG_TYPE('callback', 'function', callback);
-    }
+function queueMicrotask(callback) {
+  if (typeof callback !== 'function') {
+    throw new ERR_INVALID_ARG_TYPE('callback', 'function', callback);
+  }
 
-    const asyncResource = new AsyncResource('Microtask', {
-      triggerAsyncId: getDefaultTriggerAsyncId(),
-      requireManualDestroy: true,
-    });
+  const asyncResource = new AsyncResource('Microtask', {
+    triggerAsyncId: getDefaultTriggerAsyncId(),
+    requireManualDestroy: true,
+  });
 
-    enqueueMicrotask(() => {
-      asyncResource.runInAsyncScope(() => {
-        try {
-          callback();
-        } catch (error) {
-          // TODO(devsnek) remove this if
-          // https://bugs.chromium.org/p/v8/issues/detail?id=8326
-          // is resolved such that V8 triggers the fatal exception
-          // handler for microtasks
-          triggerFatalException(error);
-        } finally {
-          asyncResource.emitDestroy();
-        }
-      });
+  enqueueMicrotask(() => {
+    asyncResource.runInAsyncScope(() => {
+      try {
+        callback();
+      } catch (error) {
+        // TODO(devsnek) remove this if
+        // https://bugs.chromium.org/p/v8/issues/detail?id=8326
+        // is resolved such that V8 triggers the fatal exception
+        // handler for microtasks
+        triggerFatalException(error);
+      } finally {
+        asyncResource.emitDestroy();
+      }
     });
-  };
-
-  return queueMicrotask;
+  });
 };
```

Comment:
Linter doesn't like the semi-colon.

```suggestion
}
```